File: app/src/main/java/com/digis/android/task/implementation/java/data/service/MainService.java
package com.digis.android.task.implementation.java.data.service;

import com.digis.android.task.implementation.java.data.model.NetInfo;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface MainService {
    @GET("netInfo/{id}")
    Call<NetInfo> getNetInfo(@Path("id") int id);
}

File: app/src/main/java/com/digis/android/task/implementation/kotlin/data/service/MainService.kt
package com.digis.android.task.implementation.kotlin.data.service

import androidx.lifecycle.LiveData
import com.digis.android.task.implementation.kotlin.data.model.NetInfo
import com.digis.android.task.implementation.kotlin.data.source.remote.Response
import retrofit2.http.GET
import retrofit2.http.Path

interface MainService {
    @GET("netInfo/{id}")
    fun getNetInfo(@Path("id") id: Int): LiveData<Response<NetInfo>>
}